src/components/Checkbox/index.css

```
@@ -0,0 +1,8 @@
+.checkbox {
```

```suggestion
input[type=checkbox] {
```
instead of styling the check boxes using `classNames` you can do this